tensorflow/core/util/mkl_util.h

```diff
@@ -1974,9 +1354,9 @@ class MklDnnData {
     return false;
   }
 
-  /// TODO: this is a faster path with reorder primitive cache compared with
-  /// CheckReorderToOpMem(..., std::vector<primitive>* net), will remove
-  /// slow path in the future
+  /// This is a faster path with reorder primitive cache compared with
+  /// CheckReorderToOpMem(..., std::vector<primitive>* net).
+  /// The slower path will be removed in the future
```

Comment:
Adding TODO for easy search.
```suggestion
  /// TODO: Remove the slower path.
```

Reply:
Will take the suggestion

Reply:
Done